Makefile

```diff
@@ -5,7 +5,6 @@ LDFLAGS?="-X main.version=${VERSION} -X main.gitCommit=${GIT_COMMIT} -X main.bui
 GO111MODULE=on
 # Docker
 IMAGE=amazon/app-mesh-controller
-REGION=$(shell aws configure get region)
```

Comment:
This is unused in the Makefile, and mislead me for a bit (my default region is different from my ENV var). Removing it.